test/servico/cst10Test.java

```diff
@@ -71,10 +65,11 @@ public void tearDown() {
     public void testeAceitarAlocação() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Professor3", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha("Professor3", "123456"));
         
         NotificacaoService notificaçãoService = new NotificacaoService();
-        List<AusenciaModel> ausencias = notificaçãoService.listarAusenciasPorSubstituto("Professor3");
+        List<AusenciaModel> ausencias = 

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def' child have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/calebepb/substituicao/file/2777960760/issues/source?bid=3364462&fileBranchId=3364484#l71)